hack/latest-kind-images.sh

```diff
@@ -54,17 +54,17 @@ LATEST_124_TAG=$(latest_kind_tag "1\\.24")
 LATEST_125_TAG=$(latest_kind_tag "1\\.25")
 LATEST_126_TAG=$(latest_kind_tag "1\\.26")
 LATEST_127_TAG=$(latest_kind_tag "1\\.27")
+LATEST_128_TAG=$(latest_kind_tag "1\\.28")
+LATEST_129_TAG=$(latest_kind_tag "1\\.29")
 
 LATEST_122_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_122_TAG)
 LATEST_123_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_123_TAG)
 LATEST_124_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_124_TAG)
 LATEST_125_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_125_TAG)
 LATEST_126_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_126_TAG)
 LATEST_127_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_127_TAG)
-
-# k8s 1.28 is manually added to ensure that we use the exact documented tag as per kind recommendation
-LATEST_128_TAG=v1.28.0
-LATEST_128_DIGEST=sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31
```

Comment:
This special case for the latest version didn't make sense.
Each Kind release documents a specific hash for every supported K8S node image,
not just the latest version.